module DeclaringException

exception SomethingIsWrong

exception Ooops = { a, b, c }
